DAQ_System/Source/Application/DataLogger/SdDataLogger.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+* SD Data Logger
+* Authors:        Cristian Cruz
+*                 Ethan Fischer
+*
+* Email:          cris14.alex@gmail.com
+*                 ejf180001@utdallas.edu
+* 
+* (c) 2022 Dallas Formula Racing - Embedded Firmware Team
+* Formula SAE International Collegiate Chapter
+* GPL-3.0 License
+*/
+
+#include "SdDataLogger.hpp"
+#include "FATFileSystem.h"
+#include "SDBlockDevice.h"
+
+namespace application {
+
+// block device and file system declarations
+SDBlockDevice block_device_;
+FATFileSystem file_system_;
+FILE* data_file_;
+
+SdDataLogger::SdDataLogger(PinName mosi, PinName miso, PinName sck, PinName cs) 
+  : block_device_(mosi, miso, sck, cs) {
+    Mount(&block_device_);
+
+    // initialize write buffer used to write to a file
+    char write_buffer[kBlockSectorByteSize];
+    memset(write_buffer, '\0', sizeof(write_buffer));
```

Comment:
Remove unused variable, `write_buffer`
Be careful not to confuse this with `write_buffer_`

Reply:
Done, caught that a bit ago just haven't pushed those changes yet.